src/components/texts/Phrase-Word.tsx

```diff
@@ -70,38 +76,50 @@ export const Word = function ({ word, dataKey }: { word: string, dataKey:string
   const markedWords = useRecoilValue(markedwordsState);
   const wordStatus = markedWords[word.toLowerCase()];
 
-  let wordClass = 'word ';
+  let wordClass = '';
 
-  if (wordStatus) wordClass += wordStatus;
+  if (wordStatus === 'learning') {
+    wordClass = 'bg-amber-500';
+  } else if (wordStatus === 'familiar') {
+    wordClass = 'bg-yellow-500';
+  }
 
   return (
-    <span onMouseUp={(event) => getWordOrPhrase(event)}
-          className={wordClass}
-          data-key={dataKey}>
-      {word}
-    </span>
+    <div className='inline-block my-1'>
+      <span onMouseUp={(event) => getWordOrPhrase(event)}
+        className={`${wordClass} cursor-pointer border border-transparent hover:border-blue-500 hover:border py-1 p-px rounded-md`}
+        data-key={dataKey}>
+        {word}
+      </span>
+    </div>
   );
 };
 
 
-export const Phrase = function ({ phrase }: { phrase: string }) {
+export const Phrase = function ({ phrase, context }: { phrase: string, context: string }) {
   const markedWords = useRecoilValue(markedwordsState);
   const phraseStatus = markedWords[phrase.toLowerCase()];
 
-  let phraseClass = 'phrase ';
+  let wordClass = '';
 
-  if (phraseStatus) phraseClass += phraseStatus;
+  if (phraseStatus === 'learning') {
+    wordClass = 'bg-amber-500';
+  } else if (phraseStatus === 'familiar') {
+    wordClass = 'bg-yellow-500';
+  }
```

Comment:
"Learned" status doesn't get a class but should.

Reply:
Also, colours need to be consistent with translation input buttons. 
And while I'm at it, I suggest a light gray for learned words.

Reply:
Yes we need to come up with a better color scheme. For now I set it to agree with the buttons, but we need to discuss this, maybe tomorrow morning at the meeting?